### DISCUSSION TITLE
git.edgedb.com

### DISCUSSION THREAD

**Discussion by dmgolembiowski** [geldata/gel#1924] [Ideas]
This might seem silly, but could I request a 302 subdomain forward from git.edgedb.com or a URI location change from edgedb.com/git to resolve to this repository?
My apologies if this is an inappropriate thing to request, but it would be enormously helpful considering the number of times I accidentally navigate to edgedb.com on a given day when I'm intending to go to the GitHub... and "github.com/edgedb/edgedb" is honestly hard to get right on the first attempt.
I promise the effort to create this record would not go wasted.

Edit: From experience, I know it's simpler to do a URI forward via [edgedb.com/git](https://github.com/edgedb/edgedb) rather than to declare a new subdomain on your domain certificate. Is this more amenable?

**Comment by agritheory** [comment]
A `docs.edgedb.com` alias would also be appreciated.
